aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/container/TabsHandler.java

```diff
@@ -99,6 +100,13 @@ public void accept(Class<?> componentClass, Element parentElement) {
                             tabFields.putIfAbsent(tab.title(), new LinkedList<>());
                         });
             }
+            if (cls.isAnnotationPresent(DesignDialog.class)) {
```

Comment:
It's no good duplicating these lines here. What if we, by a chance, have both annotations present? I'd rather extracted a "suppressing" set of tabs and then managed it, like `Tab[] tabs = Optional.ofNullable(cls.getAnnotation(Dialog.class)).map(Dialog::tabs).orElse(cls.getAnnotation(DesignDialog.class).map(DesignDialog::tabs)); if (tabs != null) {...}` (code is simplified here)